tests/test_optim.py

```
@@ -599,3 +601,432 @@ def test_minuit_all_fixed_params():
     data = [80] + model.config.auxdata
     result = pyhf.infer.mle.fit(data, model, fixed_params=[True, True])
     assert result is not None
+
+
+def _warning_lines(caplog, match="fit result for parameter"):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_warning_lines`

(ANN202)